install.php

```diff
@@ -1610,38 +1633,70 @@ function BuildFileList(){
 <a href="?dept&preflight-ok"><li>Departments</li></a>
 <a href="?dc&preflight-ok"><li>Data Centers</li></a>
 <a href="?cab&preflight-ok"><li>Cabinets</li></a>
-<a><li class="active">LDAP</li></a>
+<?php if ( AUTHENTICATION == "LDAP" ) echo '<a><li class="active">LDAP</li></a>'; ?>
+<?php if ( AUTHENTICATION == "AD" ) echo '<a><li class="active">AD</li></a>'; ?>
 <?php if(isset($complete)){ echo '<a href="?complete&preflight-ok"><li>Complete</li></a>'; }?>
 </ul>
 </div>
 <div class='main'>
 <h2><?php echo $config->ParameterArray['OrgName']; ?></h2>
-<h3>Installation Complete</h3>
-<?php echo $nodccab; ?>
+<h3>Authentication and Authorization</h3>
+<?php
+if (!isset($_SESSION['ldapauthset'])) {
+	echo '<h3>Configure and check settings, then press Set.</h3>';
+}
+?>
 <div class='center'><div>
 <form action="<?php echo $_SERVER['SCRIPT_NAME']; ?>?ldap&preflight-ok" method="POST">
+<div id="ldap">
 <?php
-echo '<div id="ldap">
-	<h3>',__("LDAP Authentication and Authorization Configuration"),'</h3>
+$ldapserver = (!isset($_SESSION['ldapserver']))?$config->ParameterArray["LDAPServer"]:$_SESSION['ldapserver'];
```

Comment:
The user enters LDAP server, LDAP Base DN and LDAP Bind DN during ldap_bootstrap.php and they get saved in to their session. So we bring them over to the install so they're not entering the same thing again.